great_expectations/validator/metrics_calculator.py

```diff
@@ -62,14 +70,24 @@ def columns(self, domain_kwargs: Optional[Dict[str, Any]] = None) -> List[str]:
 
         return columns
 
+    @public_api
     def head(
         self,
         n_rows: int = 5,
         domain_kwargs: Optional[Dict[str, Any]] = None,
         fetch_all: bool = False,
     ) -> pd.DataFrame:
-        """
-        Convenience method to run "table.head" metric.
+        """Convenience method to return the first several rows or records from a Batch of data.
+
+        (To be deprecated in favor of using methods in "MetricsCalculator" class.)
```

Comment:
I think we have an `@deprecated_method_or_class` decorator to mark things as deprecated. I haven't used it but you can find examples in the code or ask @anthonyburdi. Using it will mark it as deprecated in our published docs.